src/sentry/runner/commands/createuser.py

```diff
@@ -76,24 +77,30 @@ def createuser(email, password, superuser, no_password, no_input):
     if password:
         user.set_password(password)
 
-    user.save()
-
-    click.echo("User created: %s" % (email,))
-
-    # TODO(dcramer): kill this when we improve flows
-    if settings.SENTRY_SINGLE_ORGANIZATION:
-        from sentry.models import Organization, OrganizationMember, OrganizationMemberTeam, Team
-
-        org = Organization.get_default()
-        if superuser:
-            role = roles.get_top_dog().id
+    if User.objects.filter(username=email).exists():
+        if force:
+            user.save(force_update=True)
+            click.echo("User updated: %s" % (email,))
         else:
-            role = org.default_role
-        member = OrganizationMember.objects.create(organization=org, user=user, role=role)
-
-        # if we've only got a single team let's go ahead and give
-        # access to that team as its likely the desired outcome
-        teams = list(Team.objects.filter(organization=org)[0:2])
-        if len(teams) == 1:
-            OrganizationMemberTeam.objects.create(team=teams[0], organizationmember=member)
-        click.echo("Added to organization: %s" % (org.slug,))
+            click.echo("User: %s exists, use --force to force an update" % (email,))
```

Comment:
This case should exit with an error for the safety I mentioned.

Reply:
Done :)